### ISSUE TITLE
Version `0.6.2` converts `/` into empty string

### ISSUE THREAD

**Issue by iiroj** [scottrippey/next-router-mock#28]
The code here also converts the literal url `"/"` into an empty string `""`:

https://github.com/scottrippey/next-router-mock/blob/8a47b4e4b3a2ebe544b6a4164d95b05e9ed945de/src/MemoryRouter.tsx#L27

This works for a minimum reproduction:

```ts
mockRouter.setCurrentUrl('/')
expect(mockRouter.asPath).toEqual('/')
```

```
expect(received).toEqual(expected) // deep equality

    Expected: "/"
    Received: ""
```

**Comment by scottrippey** [comment]
Thanks for reporting.  Seems like anything that ends with `/` would be incorrectly removed.  Will work on a fix!

**Comment by scottrippey** [comment]
As a temporary workaround, `0.6.1` does not include this behavior.  `0.6.2` introduced "catch all" slug support, so if that's not needed, rollback for  now.

**Comment by scottrippey** [comment]
According to [the Next docs about trailing slashes](https://nextjs.org/docs/api-reference/next.config.js/trailing-slash), it looks like Next's default behavior is to omit the trailing slashes.
So, I think this behavior actually seems valid for the default config.  So perhaps this issue only exists if your config has  `trailingSlash: true`?

**Comment by iiroj** [comment]
When `window.location.pathname === "/"`, then also `router.asPath === "/"`, at least with the real implementation of `next/router` version `11.1.2`. I would assume this mock would behave the same way. I'm not using the `trailingSlash` setting.

**Comment by scottrippey** [comment]
You're right ... for the empty path scenario, there should always be a `"/"`. It's not a "trailing slash", it's a "starting slash".  
I'll push a fix soon!

**Comment by scottrippey** [comment]
Fixed in `0.6.3`!

**Comment by iiroj** [comment]
@scottrippey thanks! I actually was also bitten by the missing hash, but didn't open an issue yet.
